Devfile.yaml

```diff
@@ -0,0 +1,48 @@
+---
+specVersion: 0.0.1
+name: petclinic-dev-environment
+projects:
+  - name: petclinic
+    source:
+      type: git
+      location: 'git@github.com:spring-projects/spring-petclinic.git'
+tools:
+  - name: mvn-stack
+    type: dockerImage
+    image: maven:3.5.4-jdk-8
+    entryPoint: sleep infinity
+    persistentVolumes:
+      - name: maven-repo
+        containerPath: /root/.m2
+    services:
+      - name: spring-boot
+        port: 8080
+    mountProjectsSources: true
+  - name: mysql
+    type: kubernetes
+    local: petclinic.yaml
```

Comment:
This is missing selectors. petclinic.yml has two containers that it creates. We need to define a selector here to select only the mysql one.

Reply:
Actually, `petclinic.yaml` contains one pod with two containers. Kubernetes containers does not have any labels. The current implementation is supposed to create all objects from the referenced file. Tool name should be changed to `petclinic-app`.

About `selector` field. It is a quite common practice to have all objects related to one application in one yaml file (with objects list or openshift template). 
I think It makes sense to consider and discuss introducing `selector` in kubernetes/openshift tool in a separate issue. Do you want me to create such an issue?

Reply:
I think what @gorkem means is that the petclinic application should not be running here since it runs in the mvn-stack.  Basically we have a workspace composed of 2 runtime containers:
- spring-boot petclinic app
- mysql

Reply:
@l0rd It makes sense for me but in this PR I prepared modification of existing `Devfile` to be consistent with implementation in Che repository. And I just wanted to say that introducing of `selector` field in Kubernetes/OpenShift tool I would make in a separate issue/PR (the corresponding changes should be done in Che repository).